qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -869,6 +876,55 @@ def errors(self) -> str:
         self.status()  # Collect new errors.
         return "\n".join(self._errors)
 
+    def copy_metadata(
```

Comment:
Should this be an internal method `_copy_metadata`?

Reply:
done in  bfd361d